includes/libraries/parsecsv.lib.php

```diff
@@ -0,0 +1,1163 @@
+<?php
+
+class parseCSV {
```

Comment:
@chriscct7 Already has a class exists inside the `__construct()` of the import class:
```php
if( ! class_exists( 'parseCSV' ) ) {
	require_once EDD_PLUGIN_DIR . 'includes/libraries/parsecsv.lib.php';
}
```